src/main/java/io/vertx/core/json/Json.java

```diff
@@ -111,6 +116,8 @@ static Object checkAndCopy(Object val, boolean copy) {
       }
     } else if (val instanceof byte[]) {
       val = Base64.getEncoder().encodeToString((byte[])val);
+    } else if (val instanceof Instant) {
+      val = OffsetDateTime.ofInstant((Instant) val, ZoneOffset.UTC).format(ISO_OFFSET_DATE_TIME);
```

Comment:
How is this related to the range PR? Maybe have some other commits mixed up in here?